src/mon/MonmapMonitor.cc

```diff
@@ -231,6 +264,43 @@ void MonmapMonitor::dump_info(Formatter *f)
 }
 
 bool MonmapMonitor::preprocess_command(MonOpRequestRef op)
```

Comment:
This is an actual command multiplexer. It sets up the environment for commands to run in and cleanly manages the result of applying a command.